execution.bs

```diff
@@ -4589,52 +4748,124 @@ enum class forward_progress_guarantee {
 
     1. [*Example:*
         <pre highlight=c++>
-        auto s1 = execution::just() | execution::then([]{});
+        auto s1 = just() | then([]{});
         using S1 = decltype(s1);
 
-        static_assert(execution::sender_of&lt;S1, execution::set_value_t()>);
-        static_assert(execution::sender_of&lt;S1, execution::set_error_t(exception_ptr)>);
-        static_assert(!execution::sender_of&lt;S1, execution::set_stopped_t()>);
+        static_assert(sender_of&lt;S1, set_value_t()>);
+        static_assert(sender_of&lt;S1, set_error_t(exception_ptr)>);
+        static_assert(!sender_of&lt;S1, set_stopped_t()>);
 
-        auto s2 = s1 | execution::let_error([](auto) { return execution::just('a'); });
+        auto s2 = s1 | let_error([](auto) { return just('a'); });
         using S2 = decltype(s2);
 
-        static_assert(!execution::sender_of&lt;S2, execution::set_value_t()>);
-        static_assert(!execution::sender_of&lt;S2, execution::set_value_t(char)>);
-        static_assert(!execution::sender_of&lt;S2, execution::set_error_t(exception_ptr)>);
-        static_assert(!execution::sender_of&lt;S2, execution::set_stopped_t()>);
+        static_assert(!sender_of&lt;S2, set_value_t()>);
+        static_assert(!sender_of&lt;S2, set_value_t(char)>);
+        static_assert(!sender_of&lt;S2, set_error_t(exception_ptr)>);
+        static_assert(!sender_of&lt;S2, set_stopped_t()>);
         </pre>
         -- *end example*]
 
-### Completion signatures <b>[exec.sndtraits]</b> ### {#spec-exec.sndtraits}
+5. Library-provided sender types:
+      - Always expose an overload of a customization of `connect`
+          that accepts an rvalue sender.
+      - Only expose an overload of a customization of `connect` that
+          accepts an lvalue sender if they model `copy_constructible`.
+      - Model `copy_constructible` if they satisfy `copy_constructible`.
+
+### Awaitable helpers <b>[exec.awaitables]</b> ### {#spec.exec-awaitables}
```

Comment:
This is a new subsection in which I try to nail down what [exec] means by "awaitable". All eyes appreciated.